requirements.txt

```diff
@@ -0,0 +1 @@
+awscli==1.7.25
```

Comment:
awscli is a python application so it is fitting to use `pip` to manage the dependency. it's a good idea to let Heroku handle fetching the dependency. less things to break.